addons/ui/config.cpp

```diff
@@ -11,6 +11,9 @@ class CfgPatches {
         version = VERSION;
         authors[] = {"Dr Eyeball", "commy2"};
     };
+
+    // Backwards compatibility
+    class cba_ui_helper: ADDON {author = "";};
```

Comment:
Why not just inherit author? It's set to CBA anyways.

Reply:
This prevents the credits line stuff from the help component to show up this removed component from time to time. Same is done for other bwc CfgPatches entries like the four aliases of XEH.

Reply:
Oh I see, time to do that in ACRE... :P
Thanks!